integration-tests/tests/fixtures/integration_test_logs.py

```diff
@@ -1,4 +1,4 @@
-"""Define test logs fixtures."""
+"""Session scoped test log fixtures shared across integration tests."""
```

Comment:
```suggestion
"""Session-scoped test log fixtures shared across integration tests."""
```
nit: should have a hyphen